account_invoice_pricelist/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Pricelist on Invoices
+=====================
+
+* Add a stored field pricelist on invoices, related to the partner pricelist;
+* Possibility to group by pricelist on account.invoice view;
+
+.. image:: static/src/description/screenshot_group_by.png
+
+
+This module doesn't add real feature by it self for end-users, but is usefull
+to do reporting, in a inherited module.
+
+Installation
+============
+
+Classical installation.
+
+Configuration
+=============
+
+Nothing to do.
+
+Note
+====
+
+* The field will not be stored for purchase invoices as long as 'purchase'
+  module is not installed; (in invoice / in refund). (because purchase
+  pricelist field on partner model is defined in 'purchase' module).
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to ...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/95/8.0
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
+======================
+
+* The pricelist field is readonly, so it's not possible to change it and to
```

Comment:
I'm not agree with you on that point. I'm not sure that the aim of OCA module is to describe Odoo feature. (or bugs).
In Odoo when we select a partner with a specific pricelist : 
- It will apply **special price** on sale module (sale.order);
- It will apply **special price** on purchase module (purchase.order);
- It will apply **standard price** on account module (account.invoice);

It's not consistent, but this is Odoo mechanism, and this module doesn't change it (for the time being).